components/kyma-environment-broker/internal/process/deprovisioning/monitoring_uninstall.go

```diff
@@ -31,6 +31,9 @@ func (s *MonitoringUnistallStep) Name() string {
 
 func (s *MonitoringUnistallStep) Run(operation internal.DeprovisioningOperation, log logrus.FieldLogger) (internal.DeprovisioningOperation, time.Duration, error) {
 	releaseName := operation.InstanceDetails.ShootName
+	if releaseName == "" {
+		return s.operationManager.OperationFailed(operation, "rmi release name cannot be empty", log)
+	}
```

Comment:
do not fail a step in the deprovisioning operation. If this logic cannot be executed, we have to log it, but go to next step, see EDP deprovisioning step